src/views/RegisterPage/index.js

```diff
@@ -327,7 +333,7 @@ const RegisterPage = () => {
                 )}
               </div>
             </div>
-            <StepTwo members={members} setMembers={setMembers} setValidMembers={setValidMembers} validMembers={validMembers} />
+            <StepTwo members={members} setMembers={setMembers} setValidMembers={setValidMembers} validMembers={validMembers} success={success} />
             <div className={css.stepOne__group}>
               <button type="submit" className={css.stepOne__submit} disabled={!validName || !validPwd || !validMatch || !validEmail || !validMembers}>
```

Comment:
nwm czy bym tego disabled nie zmienil w buttonie tutaj, na np mala funkcje ktora by sprawdzala

Reply:
Kiedy przycisk jest już aktywny i ktoś go kliknie, to wykonuje się funkcja, która najpierw sprawdza czy jest wszystko poprawnie wypełnione na wszelki wypadek